### ISSUE TITLE
Why did I test the program's inference speed on the computer and achieve 50s instead of the 40ms mentioned in the article?

### ISSUE THREAD

**Issue by amazingpanpanda** [Pointcept/PointTransformerV3#92]
![1725962698758](https://github.com/user-attachments/assets/d18ac98f-7fe9-430c-9377-02529d957823)

**Comment by Gofinge** [comment]
Your local machine is 4090?

**Comment by amazingpanpanda** [comment]
> Your local machine is 4090?

A100

**Comment by Gofinge** [comment]
> > Your local machine is 4090?
> 
> A100

Wow. Do do the inference latency testing with training script, comment code for backward.

**Comment by Gofinge** [comment]
> ![1725962698758](https://private-user-images.githubusercontent.com/66508286/365996312-d18ac98f-7fe9-430c-9377-02529d957823.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYwMzIwOTUsIm5iZiI6MTcyNjAzMTc5NSwicGF0aCI6Ii82NjUwODI4Ni8zNjU5OTYzMTItZDE4YWM5OGYtN2ZlOS00MzBjLTkzNzctMDI1MjlkOTU3ODIzLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTExVDA1MTYzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiYTVmZTc4ZTM0OTVhOGYxNDUwMWY3YjU2OGU0MjA5MzZjZjUzMDAxYTFiMGUwZmI0ZGFiMjlhMzgwNDEwMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CmsghSb9ekmNN9rNvQLZlBFcKVJuAOGhPz1icIyZjik)

Oh I know your meaning. Inference latency means the time for one forward. The test script was forwarded multiple times to get the best performance.

**Comment by amazingpanpanda** [comment]
> > > Your local machine is 4090?
> > 
> > 
> > A100
> 
> Wow. I tested the inference latency testing with our training script and comment code for backward.

So may I ask if the "44ms" mentioned in the third part of Figure 1 of the paper by Faster Speed refers to the time required for backbone inference, rather than the time required for the model to infer a frame of data?

**Comment by Gofinge** [comment]
Actually, inference one frame one is sufficient for the application. (Simple interpolating the prediction to the original resolution also performs well). The complex testing process is just for getting a higher number.

**Comment by amazingpanpanda** [comment]
> Actually, inference one frame one is sufficient for the application. (Simple interpolating the prediction to the original resolution also performs well). The complex testing process is just for getting a higher number.

May I ask how long it takes for you to infer a frame of data during testing?

**Comment by Gofinge** [comment]
Do you mean the current default test script?

**Comment by amazingpanpanda** [comment]
> Do you mean the current default test script?

yes

**Comment by Gofinge** [comment]
> > Do you mean the current default test script?
> 
> yes

I think I should be the same as you. As I usually do testing automatically after training (with 8 GPUs), I didn't measure the time and just wait the training end.

**Comment by amazingpanpanda** [comment]
> > > Do you mean the current default test script?
> > 
> > 
> > yes
> 
> I think I should be the same as you. As I usually do testing automatically after training (with 8 GPUs), I didn't measure the time and just wait the training end.

OK, thank you for your answer.

**Comment by Gofinge** [comment]
> > > > Do you mean the current default test script?
> > > 
> > > 
> > > yes
> > 
> > 
> > I think I should be the same as you. As I usually do testing automatically after training (with 8 GPUs), I didn't measure the time and just wait the training end.
> 
> OK, thank you for your answer.

If you want to save more time, you can reduce the number of augmentation for testing.

**Comment by amazingpanpanda** [comment]
> > > > > Do you mean the current default test script?
> > > > 
> > > > 
> > > > yes
> > > 
> > > 
> > > I think I should be the same as you. As I usually do testing automatically after training (with 8 GPUs), I didn't measure the time and just wait the training end.
> > 
> > 
> > OK, thank you for your answer.
> 
> If you want to save more time, you can reduce the number of augmentation for testing.

Yes, I found through testing that even with only one enhancement, its inference time can reach 200ms to 400ms.

**Comment by Gofinge** [comment]
Usually, it is good enough. But you know, for benchmarking, we are usually willing to spend more testing time for even 0.1% enhancement.

**Comment by amazingpanpanda** [comment]
> Usually, it is good enough. But you know, for benchmarking, we are usually willing to spend more testing time for even 0.1% enhancement.

OK
